src/Iris.php

```diff
@@ -54,16 +53,32 @@ public function __construct()
         add_action('plugins_loaded', [__CLASS__, 'check_for_offload_media']);
 
         // IRIS :)
-        add_filter('wp_generate_attachment_metadata', [__CLASS__, 'iris_webp_converter'], 10, 3);
+        add_filter('wp_generate_attachment_metadata', [__CLASS__, 'irisWebpConverter'], 10, 3);
+        add_filter('wp_generate_attachment_metadata', [__CLASS__, 'iris'], 10, 3);
```

Comment:
Same as above, ideally this method should provide some context to its purpose, but currently it's not clear what the `iris` method could be doing?